lca-chimevc-stack/lambda_functions/pca_integration/index.js

```diff
@@ -109,36 +109,55 @@ const writePcaUrlToKds = async function writePcaUrlToKds(kinesisClient, sessionD
   }
 }
 
-const copyAudioRecordingToPca = async function copyAudioRecordingToPca(s3Client, sessionData) {
+const getS3PathFromUri = function getS3SourceUri(s3Uri) {
```

Comment:
Current `process_tca_postcall` only processes post call categories. But this would seem to be the logical place to add a call to `writeS3UrlToKds` as well to display the recording URL for playback, now that it is actually available.